src/features/VoltageConfiguration/VoltageConfiguration.tsx

```diff
@@ -21,24 +21,36 @@ interface VoltageConfigurationProps {
     pmicPort: number;
     voltageMin: number;
     voltageMax: number;
+    pmicPortLabel?: string;
+    pmicPortDescription?: string;
 }
 
 const VoltageConfiguration = ({
     pmicPort,
     voltageMin,
     voltageMax,
+    pmicPortLabel,
+    pmicPortDescription,
 }: VoltageConfigurationProps) => {
     const dispatch = useDispatch();
     const voltage = useSelector(getPmicConfigValue(pmicPort)) ?? voltageMin; // Default to voltageMin
 
+    const label = pmicPortLabel ?? `Voltage port ${pmicPort}`;
+    const description =
+        pmicPortDescription ??
+        `Set voltage for PMIC port ${pmicPort} (${label})`;
+
     return (
         <Card
             title={
                 <div className="tw-flex tw-content-between">
-                    <span>Voltage</span>
+                    <span>{label}</span>
                 </div>
             }
         >
+            <div className="tw-flex tw-gap-1">
```

Comment:
Same question about use of flex here

Reply:
Same answer

Reply:
At least here the surrounding div was added. I would be really curious to see if removing the outer div changes anything at all.
Given that there is just 1 child I actually can't see that being the case.
Also the gap-1 here is definitely unused since that will only be relevant for 2 or more children.